kubernetes/elastic-helm/values.yaml

```
@@ -4,6 +4,12 @@ default:
     tag: 1.10.1
 
 opentelemetry-collector:
+  image:
+    repository: docker.elastic.co/beats/elastic-agent
+    tag: 8.15.0-7b611e39-SNAPSHOT
```

```suggestion
    tag: 8.15.0-SNAPSHOT
```
Maybe just point to the latest SNAPSHOT for now? Eventually we should switch to `8.15.0`.